src/www/app/outline_server_repository/access_key_serialization.ts

```diff
@@ -70,7 +70,7 @@ function parseShadowsocksSessionConfigJson(maybeJsonText: string): ShadowsocksSe
 export async function fetchShadowsocksSessionConfig(configLocation: URL): Promise<ShadowsocksSessionConfig> {
   let response;
   try {
-    response = await fetch(configLocation, {cache: 'no-store'});
+    response = await fetch(configLocation, {cache: 'no-store', redirect: 'follow'});
```

Comment:
This is already the default behavior, so this won't be fixing any issue. I guess the goal is clarity?
https://developer.mozilla.org/en-US/docs/Web/API/Request/redirect

Reply:
Rename the PR to refactor, since this is not a fix (no change in behavior).